src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -0,0 +1,134 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ===================================
+# Schema for:
+#  * 'show wireless mobility summary'
+# ===================================
+class ShowWirelessMobilitySummarySchema(MetaParser):
+    """Schema for show wireless mobility summary."""
+
+    schema = {
+        "controller_config": {
+            "group_name": str,
+            "ipv4": str,
+            "mac_address": str,
+            "multicast_ipv4": str,
+            "multicast_ipv6": str,
+            "pmtu": str,
+            "public_ip": str,
+            "status": str,
+        },
+        "mobility_summary": {
+            "domain_id": str,
+            "dscp_value": str,
+            "group_name": str,
+            "keepalive": str,
+            "mac_addr": str,
+            "mgmt_ipv4": str,
+            "mgmt_ipv6": str,
+            "mgmt_vlan": str,
+            "multi_ipv4": str,
+            "multi_ipv6": str,
+        },
+    }
+
+
+# ===================================
+# Parser for:
+#  * 'show wireless mobility summary'
+# ===================================
+class ShowWirelessMobilitySummary(ShowWirelessMobilitySummarySchema):
+    """Parser for show wireless mobility summary"""
+
+    cli_command = ["show wireless mobility summary"]
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command[0])
```

Comment:
won't need to index this when above is changed

Reply:
Done!